src/web/services/log.service.ts

```diff
@@ -31,4 +32,21 @@ export class LogService {
     return this.httpRequestService.post(ResourceEndpoints.TRACK_SESSION, paramMap);
   }
 
+  /**
+   * Searches for feedback session logs.
+   */
+  searchFeedbackSessionLog(queryParams: {
+    courseId: string,
+    searchFrom: string,
+    searchUntil: string
+    studentEmail?: string,
+  }): Observable<FeedbackSessionLogs> {
+    const paramMap: Record<string, string> = {
+      courseid: queryParams.courseId,
+      fslstarttime: queryParams.searchFrom,
+      fslendtime: queryParams.searchUntil,
+    };
```

Comment:
Is there any way to use the student email in the params here (even though it's optional)?